common/src/main/java/io/druid/guice/DruidSecondaryModule.java

```diff
@@ -75,24 +76,36 @@ public void configure(Binder binder)
     binder.bind(JsonConfigurator.class).toInstance(jsonConfigurator);
   }
 
-  @Provides @LazySingleton @Json
+  @Provides
+  @LazySingleton
+  public InjectableValues getGuiceInjectableValues(final Injector injector)
+  {
+    return new GuiceInjectableValues(injector);
+  }
+
+  @Provides
+  @LazySingleton
+  @Json
   public ObjectMapper getJsonMapper(final Injector injector)
   {
     setupJackson(injector, jsonMapper);
     return jsonMapper;
   }
 
-  @Provides @LazySingleton @Smile
+  @Provides
+  @LazySingleton
+  @Smile
   public ObjectMapper getSmileMapper(Injector injector)
   {
     setupJackson(injector, smileMapper);
     return smileMapper;
   }
 
-  private void setupJackson(Injector injector, final ObjectMapper mapper) {
+  private void setupJackson(Injector injector, final ObjectMapper mapper)
+  {
     final GuiceAnnotationIntrospector guiceIntrospector = new GuiceAnnotationIntrospector();
 
-    mapper.setInjectableValues(new GuiceInjectableValues(injector));
+    mapper.setInjectableValues(injector.getInstance(InjectableValues.class));
```

Comment:
I think that the changes in this file are not actually effecting any functional change.  I think it was probably done as a first attempt at trying to figure out what was wrong, but then later the issue with the copy() method on ObjectMapper was noticed?